jazz_deployments/index.js

```diff
@@ -159,15 +159,13 @@ module.exports.handler = (event, context, cb) => {
 		.catch((error) => {
 			if (error.result === "inputError") {
 				return cb(JSON.stringify(errorHandler.throwInputValidationError(error.message)));
-			} else if (error.result === "unauthorized") {
-				return cb(JSON.stringify(errorHandler.throwUnauthorizedError(error.message)));
 			} else {
 				return cb(JSON.stringify(errorHandler.throwInternalServerError("Unexpected error occurred.")))
 			}
 		});
 };
 
-function genericInputValidation(event) {
+var genericInputValidation = (event) => {
```

Comment:
why did you change all these to be function expression. you could have just exported functions

Reply:
Worked as per comments